alcs-frontend/src/app/features/card-detail-dialog/card-detail-dialog.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { UserDto } from '../../services/user/user.dto';
+import { UserService } from '../../services/user/user.service';
+import { ApplicationDto, ApplicationPartialDto } from '../application/application.dto';
+import { ApplicationService } from '../application/application.service';
+
+@Component({
+  selector: 'app-card-detail-dialog',
+  templateUrl: './card-detail-dialog.component.html',
+  styleUrls: ['./card-detail-dialog.component.scss'],
+})
+export class CardDetailDialogComponent implements OnInit {
+  assignees: UserDto[] = [];
+  filteredAssignees?: UserDto[] = [];
+  selectedAssignee?: UserDto;
+  selectedAssigneeEmail: string = '';
+  currentCard: ApplicationPartialDto = this.data;
+
+  constructor(
+    @Inject(MAT_DIALOG_DATA) public data: ApplicationDto,
+    private readonly userService: UserService,
+    private applicationService: ApplicationService
+  ) {}
+
+  ngOnInit(): void {
+    this.currentCard = this.data;
+    this.selectedAssignee = this.data.assignee;
+    this.selectedAssigneeEmail = this.selectedAssignee?.email || '';
+    this.userService.$users.subscribe((users) => {
+      this.assignees = users;
+      this.filteredAssignees = this.assignees;
+    });
+
+    this.userService.fetchUsers();
+  }
+
+  filterAssigneeList(event: any) {
```

Comment:
Change type from any to KeyboardEvent?
https://developer.mozilla.org/en-US/docs/Web/API/Event